src/Index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="libs/fontawesome/all.min.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@200;700;900&display=swap" rel="stylesheet">
+    <title>ToDOList</title>
+    <link rel="stylesheet" href="../src/css/style.css">
+    <script defer src="../src/js/script.js"></script>
+</head>
+<body> 
+    <div class="todo__list">
+        <h1 class="header-style">ToDoList</h1>
+        <div class="header__input">
```

Comment:
You use different spacing for the styles names, word style is unnecessary, and the h1 element is a single element so you can style it by tag.

Reply:
deleted class and added style it by tag